spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitConsumerProperties.java

```diff
@@ -0,0 +1,152 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.binder.rabbit;
+
+import org.springframework.amqp.core.AcknowledgeMode;
+import org.springframework.cloud.stream.binder.ConsumerProperties;
+import org.springframework.util.Assert;
+
+/**
+ * @author Marius Bogoevici
+ */
+public class RabbitConsumerProperties extends ConsumerProperties {
+
+	private String prefix = "binder.";
+
+	private boolean transacted = false;
+
+	private AcknowledgeMode acknowledgeMode = AcknowledgeMode.AUTO;
+
+	private int maxConcurrency = 1;
+
+	private int prefetch = 1;
+
+	private String[] requestHeaderPatterns = new String[] {"STANDARD_REQUEST_HEADERS", "*"};
+
+	private int txSize = 1;
+
+	private boolean autoBindDlq = false;
+
+	private boolean durableSubscription = true;
+
+	private boolean republishToDlq = false;
+
+	private boolean requeueRejected = true;
+
+	private String replyHeaderPatterns;
+
+	public String getPrefix() {
+		return prefix;
+	}
+
+	public void setPrefix(String prefix) {
+		this.prefix = prefix;
+	}
+
+	public boolean isTransacted() {
+		return transacted;
+	}
+
+	public void setTransacted(boolean transacted) {
+		this.transacted = transacted;
+	}
+
+	public AcknowledgeMode getAcknowledgeMode() {
+		return acknowledgeMode;
+	}
+
+	public void setAcknowledgeMode(AcknowledgeMode acknowledgeMode) {
+		Assert.notNull("Acknowledge mode cannot be null");
+		this.acknowledgeMode = acknowledgeMode;
+	}
+
+	public int getMaxConcurrency() {
+		return maxConcurrency;
+	}
+
+	public void setMaxConcurrency(int maxConcurrency) {
+		this.maxConcurrency = maxConcurrency;
+	}
+
+	public int getPrefetch() {
+		return prefetch;
+	}
+
+	public void setPrefetch(int prefetch) {
+		this.prefetch = prefetch;
+	}
+
+	public String[] getRequestHeaderPatterns() {
+		return requestHeaderPatterns;
+	}
+
+	public void setRequestHeaderPatterns(String[] requestHeaderPatterns) {
+		this.requestHeaderPatterns = requestHeaderPatterns;
+	}
+
+	public int getTxSize() {
+		return txSize;
+	}
+
+	public void setTxSize(int txSize) {
+		this.txSize = txSize;
+	}
+
+	public boolean isAutoBindDlq() {
+		return autoBindDlq;
+	}
+
+	public void setAutoBindDlq(boolean autoBindDlq) {
+		this.autoBindDlq = autoBindDlq;
+	}
+
+	public boolean isDurableSubscription() {
+		return durableSubscription;
+	}
+
+	public void setDurableSubscription(boolean durableSubscription) {
+		this.durableSubscription = durableSubscription;
+	}
+
+	public boolean isRepublishToDlq() {
+		return republishToDlq;
+	}
+
+	public void setRepublishToDlq(boolean republishToDlq) {
+		this.republishToDlq = republishToDlq;
+	}
+
+	public boolean getRequeueRejected() {
```

Comment:
Couple of getters for this property name. Could be better if we use `isRequeueRejected` and remove this method.